.gitignore

```diff
@@ -13,6 +13,7 @@
 *_embed.go
 gapir.log
 gapis.log
+*.pb.go
```

Comment:
Thus far, the pattern has been to add the file by name to a .gitignore file in the same dir.